### PR TITLE
ATB-1512: Update TxMA Egress tests 

### PR DESCRIPTION
**Pull request by BhanusriNarne**
### Issue tracking

- [ATB-1512](https://govukverify.atlassian.net/browse/ATB-1512)

## Testing
Screenshot 2024-03-04 at 13 22 03

Screenshot 2024-03-04 at 13 22 17

## Checklists
- [x] Did not commit any not-required changes to the `src/infra/**/samconfig.toml`
- [x] Tested changes and included test evidence in the PR, if appropriate
- [ ] Included all required tags and other properties for any new resources in the SAM template
- [ ] Ensured that any new resources in the SAM Template follow appropriate naming conventions
- [ ] Ensured that naming of new resources is compatible with deploying multiple stacks with custom stack names during development
- [ ] Ensured that no log lines include PII or other sensitive data
- [ ] Implemented unit testing for any new logic implemented, if appropriate
- [ ] Ensured that all commits in this PR are signed
- [ ] Ensured appropriate code coverage is maintained by unit tests
- [ ] Checked SonarCube and ensured no code smells were added

[ATB-1512]: https://govukverify.atlassian.net/browse/ATB-1512?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by TaherKapasi — APPROVED** [review]

File: stack-orchestration/configuration/di-id-reuse-core-build-admin/ais-main-pipeline/parameters.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
   {
     "ParameterKey": "CustomKmsKeyArns",
-    "ParameterValue": "arn:aws:kms:eu-west-2:216552277552:key/4bc58ab5-c9bb-4702-a2c3-5d339604a8fe,arn:aws:kms:eu-west-2:688341086169:key/d357b3a0-6129-46bc-9b64-6ab9ae155007,arn:aws:kms:eu-west-2:688341086169:key/ced952f4-921e-4233-b209-7481ca55e532"
+    "ParameterValue": "arn:aws:kms:eu-west-2:216552277552:key/4bc58ab5-c9bb-4702-a2c3-5d339604a8fe,arn:aws:kms:eu-west-2:688341086169:key/d357b3a0-6129-46bc-9b64-6ab9ae155007,arn:aws:kms:eu-west-2:688341086169:key/ced952f4-921e-4233-b209-7481ca55e532,arn:aws:kms:eu-west-2:688341086169:key/ced952f4-921e-4233-b209-7481ca55e532"
```
Comment (MinaMahdi) [inline stack-orchestration/configuration/di-id-reuse-core-build-admin/ais-main-pipeline/parameters.json:32]: Hey! Would you be able to please also remove the last ARN - `arn:aws:kms:eu-west-2:688341086169:key/ced952f4-921e-4233-b209-7481ca55e532` as its a duplicate, I've also removed it from the pipeline parameters in build now too - thank you 🙏

File: stack-orchestration/configuration/di-id-reuse-core-dev-admin/ais-main-pipeline/parameters.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
   },
   {
     "ParameterKey": "CustomKmsKeyArns",
-    "ParameterValue": "arn:aws:kms:eu-west-2:216552277552:key/4bc58ab5-c9bb-4702-a2c3-5d339604a8fe, arn:aws:kms:eu-west-2:013758878511:key/ab396761-f7f0-420f-afa9-7a407896913f"
+    "ParameterValue": "arn:aws:kms:eu-west-2:216552277552:key/4bc58ab5-c9bb-4702-a2c3-5d339604a8fe, arn:aws:kms:eu-west-2:013758878511:key/ab396761-f7f0-420f-afa9-7a407896913f,arn:aws:kms:eu-west-2:013758878511:key/99255faa-1e7b-4cef-b32c-a426f565ecc7git "
```
[the remark this answers is not in the document]
Comment (BhanusriNarne) [reply stack-orchestration/configuration/di-id-reuse-core-dev-admin/ais-main-pipeline/parameters.json]: removed

File: stack-orchestration/configuration/di-id-reuse-core-build-admin/ais-main-pipeline/parameters.json (hunk 1 above)
Comment (BhanusriNarne) [reply stack-orchestration/configuration/di-id-reuse-core-build-admin/ais-main-pipeline/parameters.json]: removed

**Review by MinaMahdi — APPROVED** [review]

**Review by ChrisBates1 — APPROVED** [review]

**Comment by BhanusriNarne** [comment]
* Update TxMA Egress tests

**Pull request merged**